Collège solidaire

lutangar opened the pull request:
Pour rappel :
> Les copains d'Albi ont créer leur statuts sur la base de ceux de **Cowork'in Montpel**  et des notres :
> https://github.com/cowork-in-montpellier/cowork-in-montpellier/blob/master/resources/files/statuts.pdf
> et sa donne ça:
> https://pad.georgette.party/laccordeon_statuts

Le brouillon pour nos statuts est quand à lui situé ici :
https://pad.georgette.party/amabla_statuts_solidaires

Sur la partie élu ou non d'un membre du collège j'ai encore un doute...

Je rajouterais qqs motivations en préambule de la proposition quand j'aurais le temps.

lutangar committed:
Première épreuve

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+## Article 11 – Collège solidaire
+
+L'association est administrée par un **collège solidaire** ou plus simplement dénomé ci-après comme le **collège**. Le college est constitué des membres actifs.
```
```suggestion
L'association est administrée par un **collège solidaire** ou plus simplement dénomé ci-après comme le **collège**. Le collège est constitué des membres actifs.
```

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+L'association est administrée par un **collège solidaire** ou plus simplement dénomé ci-après comme le **collège**. Le college est constitué des membres actifs.
+
+Les membres du **collège** représente l'association dans tous les actes de la vie civile. Il a, notamment, qualité pour ester en justice au nom de l'association.
```
```suggestion
Les membres du **collège** représentent l'association dans tous les actes de la vie civile. Il a, notamment, qualité pour ester en justice au nom de l'association.
```

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+Le **collège** désigne un ou plusieurs membres en son sein, qui seront délégataires de la signature sur le compte bancaire.
+Les membres qui ont la délégation de la signature sur le compte bancaire sont chargées de tenir ou faire tenir sous son contrôle la comptabilité de l’association.
+Ils effectuent tous paiements et reçoit, sous la surveillance d'un ou plusieurs membres du **collège**, toutes sommes dues à l’association.
```
```suggestion
Ils effectuent tous paiements et reçoivent, sous la surveillance d'un ou plusieurs membres du **collège**, toutes sommes dues à l’association.
```

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+Les membres qui ont la délégation de la signature sur le compte bancaire sont chargées de tenir ou faire tenir sous son contrôle la comptabilité de l’association.
+Ils effectuent tous paiements et reçoit, sous la surveillance d'un ou plusieurs membres du **collège**, toutes sommes dues à l’association.
+Ils ne peuvent aliéner les valeurs constituant le fonds de réserve qu’avec l’autorisation des autre membres du **collège**.
+Ils tiennent une comptabilité régulière de toutes les opérations qu'elles effectues et rend compte à **l’Assemblée Générale** annuelle qui approuve sa gestion.
```
```suggestion
Ils tiennent une comptabilité régulière de toutes les opérations qu'ils effectuent et rendent compte à **l’Assemblée Générale** annuelle qui approuve leur gestion.
```

Kromalin commented on README.md:
```diff
@@ -19,37 +20,37 @@ L'objectif de cet espace est de:
 
 Ils sont membres de l'assemblée générale avec voix délibérative.
 
-Sont appelés **Membres actifs** les personnes physiques adhérentes de l'association utilisant de manière régulière l'espace de travail de l'association comme environnement de travail. 
```
souris

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+En cas d'absence, le **collège** pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
+
+<!-- Prévoir ou non un vote pour élire les membres du collège? -->
```
Le but étant d'impliquer tout le monde, n'est-il pas possible d'intégrer les nouveaux membres permanents dès le prochain CA ?
Faut-il attendre une AG ?

lutangar replied:
Je pense également que le plus tôt le mieux, notre CA actuel intègre les nouveaux "permanent" comme étant membre d'office, nous pourrions procéder de la même manière pour le collège. Cependant dans ce fonctionnement je ne vois comment intégré un/des nomades au sein du collège.

lutangar replied:
Après discussion lors de l'AG il est convenu que le collège solidaire est composé des membres permanents.

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+Les membres du **collège** sont élus pour une durée de un an et les membres sortants sont rééligibles.
 
-**Le Secrétaire** est chargé de tout ce qui concerne la correspondance et les archives. Il rédige, diffuse et archive les procès verbaux des réunions et assemblées et, en général, toutes les écritures concernant le fonctionnement de l’association, à l’exception de celles qui concernent la comptabilité. Il tient le registre spécial prévu par l’article 5 de la loi du 1er juillet 1901 et les article ...
+Il est convenu qu'un minimum de 3/5e des sièges du **collège** est réservé aux membres actifs.
```
Donc 2/5 (4 membres) peuvent être des nomades ?

lutangar replied:
A nous de trouver la formule, peut-être 4/5 ou encore 9/10 ?
Comment souhaitons nous représenter les nomades? Peut-être sont-ils uniquement membre et representé uniquement lors d'une AG?

dolemoine replied:
effectivement la question des nomades n'est pas évidente. Je vois quand même difficilement comment les nomades peuvents être impliqués au sein du collège compte tenu du fait que leur présence est aléatoire et irrégulière. Peut être qu'on pourrait définir un genre de formule hybride de "nomade permanent", une offre pour quelqu'un⋅e qui veut 5 ou 10 jours de présence par mois par ex, sans avoir un bureau fixe, et qui pourrait à ce titre "représenter" la voix des nomades plus largement?

feugy replied:
Après AGE: cette notion est caduque

outofrange-consulting commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-Le décès 
-La radiation, prononcée par le Conseil d’Administration pour non-paiement de la cotisation. 
-L'exclusion, prononcée par le Conseil d’Administration pour infraction aux présents statuts ou pour tout autre motif portant préjudice aux intérêts moraux et matériels de l'association.
+*La qualité de membre se perd par:
```
```suggestion
La qualité de membre se perd par :
```

outofrange-consulting reviewed (changes requested):
Plusieurs corrections de conjugaison ou de mise en forme.
2-3 questions concernant l'éligibilité des membres.

lutangar committed:
Update README.md

Co-authored-by: Geoffrey Marc <7069588+kalanegos@users.noreply.github.com>

lutangar committed:
Update README.md

Co-authored-by: Geoffrey Marc <7069588+kalanegos@users.noreply.github.com>

lutangar committed:
Update README.md

Co-authored-by: Geoffrey Marc <7069588+kalanegos@users.noreply.github.com>

lutangar committed:
Update README.md

Co-authored-by: Geoffrey Marc <7069588+kalanegos@users.noreply.github.com>

lutangar committed:
Update README.md

Co-authored-by: Geoffrey Marc <7069588+kalanegos@users.noreply.github.com>

lutangar commented on README.md:
```diff
@@ -138,39 +158,11 @@ La bonne tenue de la comptabilité échoue au trésorier qui est le garant de l'
 - le changement de titre de l’association,
 - le transfert du siège social,
-- les changements de membres du bureau et conseil d’administration,
+- les changements de membres du **collège**,
```
Il faudrait vérifier la nécessité de déclarer les changement de membres...

feugy replied:
Après AGE: le college n'est consitué que des membres permaments, la question de la mise à jour n'a pas sens

tobika commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+En cas d'absence, le **collège** pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
+
+<!-- Prévoir ou non un vote pour élire les membres du collège? -->
+Le **collège** est composé d’un maximum de 10 membres élus par l'Assemblée Générale. En cas de nombre insuffisant de candidats (inférieur ou égal à 10), le nombre de membres du **collège** peut être limité.
```
10 c'est car on essaye d’être 10 permanents?

lutangar replied:
Oui je pense, mais nous pouvons adapter la formulation à quelque chose de variables?
Ou encore mieux supprimer ce maximum tout simplement...

tobika commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-L'association est dirigée par un bureau d'au moins deux membres, élus pour une année par l'assemblée générale. Les membres sont rééligibles. L’assemblée choisit en son sein, au scrutin secret, un bureau composé de :
+Le nombre de votants doit correspondre au moins à 1/3 des membres actifs à jour de cotisation au jour d'envoi de la première Convocation à l'Assemblée Générale.
+
+Les membres sont élus par un vote à bulletin secret dépouillé de manière transparente. En cas d'ex-aequo, les membres en ballotage seront départagés à main levée par l'Assemblée.
```
Donc chaque permanent dois se faire elire dans le college? Juste pour que je comprend bien :)

lutangar replied:
Ou alors décider que c'est automatique pour les permanents ! Et a ce moment la on peut supprimer cette partie...
Mais reste la question de la representation des nomades dans le collège....

dolemoine commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+L'association est administrée par un **collège solidaire** ou plus simplement dénomé ci-après comme le **collège**. Le collège est constitué des membres actifs.
+
+Les membres du **collège** représentent l'association dans tous les actes de la vie civile. Il a, notamment, qualité pour ester en justice au nom de l'association.
```
```suggestion
Les membres du **collège** représentent l'association dans tous les actes de la vie civile. Ils ont, notamment, qualité pour ester en justice au nom de l'association.
```

dolemoine commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-L'association est dirigée par un bureau d'au moins deux membres, élus pour une année par l'assemblée générale. Les membres sont rééligibles. L’assemblée choisit en son sein, au scrutin secret, un bureau composé de :
+Le nombre de votants doit correspondre au moins à 1/3 des membres actifs à jour de cotisation au jour d'envoi de la première Convocation à l'Assemblée Générale.
+
+Les membres sont élus par un vote à bulletin secret dépouillé de manière transparente. En cas d'ex-aequo, les membres en ballotage seront départagés à main levée par l'Assemblée.
```
est-ce que la seconde partie de ce point est toujours valable compte tenu de ce que tu as mentionné @lutangar > `Je pense également que le plus tôt le mieux, notre CA actuel intègre les nouveaux "permanent" comme étant membre d'office, nous pourrions procéder de la même manière pour le collège. Cependant dans ce fonctionnement je ne vois comment intégré un/des nomades au sein du collège.`

lutangar replied:
Effectivement, si l'on est tous d'accords la dessus ont peu simplifier 
> cc @tobika qui avait une question similaire https://github.com/Amabla/statuts-coworking-associatif/pull/2#discussion_r872503816

lutangar replied:
La simplification décidé permet de remettre la question à plus tard.

dolemoine commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-Le conseil d’administration se réunit au moins une fois par trimestre, sur convocation du Président ou sur la demande du quart de ses membres. Les décisions sont prises à la majorité des voix; en cas de partage, la voix du président est prépondérante. Tout membre du conseil qui, sans excuses, n'aura pas assisté à trois réunions consécutives, pourra être considéré comme démissionnaire.
+## Article 12 – Réunion et pouvoir du **collège**
+Le **collège** se réunit au moins une fois par trimestre, selon un calendrier annuel défini dans le réglement intérieur. Les décisions sont prises à la majorité des voix.
+Tout membre du **collège** qui, sans excuses, n'aura pas assisté à trois réunions consécutives, pourra être considéré comme démissionnaire.
```
_peut être à intégré plutôt dans l'article 11 (ou 10) qui défini le collège solidaire_
```suggestion
Tous les membres du **collège** sont sur le même pied d’égalité : chacun des membres élus est ainsi co-président de l’association et la prise de décision est effectué sur le principe d'une personne, une voix.
Tout membre du **collège** qui, sans excuses, n'aura pas assisté à trois réunions consécutives, pourra être considéré comme démissionnaire.
```

dolemoine commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+Chaque modification du Règlement Intérieur doit être validé par la majorité des 2/3 du **collège**. Ces changements doivent être approuvé en Assemblée générale. Les modifications apportées au cours de l'année par le **collège** doivent être approuvées au cours de l'Assemblée générale suivante (ordinaire ou extraordinaire).
 
 ## Article 16 – Organisation Comptable
 Il est tenu à jour une comptabilité en recettes et dépenses pour l'enregistrement de toutes opérations financières.
```
a priori il n'y aura plus de trésorier dans le cadre d'un collège solidaire, voir comment traiter cette question.

lutangar replied:
Traité dans l'article 11 : https://github.com/Amabla/statuts-coworking-associatif/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R97-R99
Es-ce que cela te semble cohérent ?

dolemoine replied:
yep nickel

lutangar committed:
Update README.md

Co-authored-by: dolemoine <43757076+dolemoine@users.noreply.github.com>

lutangar committed:
Intégre l'article 12 à l'article 10

Co-authored-by: dolemoine <43757076+dolemoine@users.noreply.github.com>

lutangar commented on README.md:
```diff
@@ -62,73 +63,93 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+En cas d'absence, le **collège** pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
+
+<!-- Prévoir ou non un vote pour élire les membres du collège? -->
+Le **collège** est composé d’un maximum de 10 membres élus par l'Assemblée Générale. En cas de nombre insuffisant de candidats (inférieur ou égal à 10), le nombre de membres du **collège** peut être limité.
```
```suggestion
Le **collège** est composé des membres permanents.
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,93 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
 
-## Article 11 – Bureau
-L'association est dirigée par un bureau d'au moins deux membres, élus pour une année par l'assemblée générale. Les membres sont rééligibles. L’assemblée choisit en son sein, au scrutin secret, un bureau composé de :
+Le nombre de votants doit correspondre au moins à 1/3 des membres actifs à jour de cotisation au jour d'envoi de la première Convocation à l'Assemblée Générale.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,93 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-L'association est dirigée par un bureau d'au moins deux membres, élus pour une année par l'assemblée générale. Les membres sont rééligibles. L’assemblée choisit en son sein, au scrutin secret, un bureau composé de :
+Le nombre de votants doit correspondre au moins à 1/3 des membres actifs à jour de cotisation au jour d'envoi de la première Convocation à l'Assemblée Générale.
+
+Les membres sont élus par un vote à bulletin secret dépouillé de manière transparente. En cas d'ex-aequo, les membres en ballotage seront départagés à main levée par l'Assemblée.
```
```suggestion
```

lutangar committed:
Le collège est composé des membres permanents.

Suppression des différents paragraphes relatifs à l'élection des membres du collèges solidaires.

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-**Le Trésorier** est chargé de tenir ou faire tenir sous son contrôle la comptabilité de l’association. Il effectue tous paiements et reçoit. sous la surveillance du Président, toutes sommes dues à l’association. Il ne peut aliéner les valeurs constituant le fonds de réserve qu’avec l’autorisation du bureau. Il tient une comptabilité régulière de toutes les opérations qu'il effectue et rend compt ...
 
-En cas d'absence, le bureau pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
+Les membres sont élus par un vote à bulletin secret dépouillé de manière transparente. En cas d'ex-aequo, les membres en ballotage seront départagés à main levée par l'Assemblée.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-En cas d'absence, le bureau pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
+Les membres sont élus par un vote à bulletin secret dépouillé de manière transparente. En cas d'ex-aequo, les membres en ballotage seront départagés à main levée par l'Assemblée.
+
+Le vote par procuration ou par correspondance est autorisé.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
 
-## Article 12 – Réunion et pouvoir du conseil d’administration
-Le conseil d’administration se réunit au moins une fois par trimestre, sur convocation du Président ou sur la demande du quart de ses membres. Les décisions sont prises à la majorité des voix; en cas de partage, la voix du président est prépondérante. Tout membre du conseil qui, sans excuses, n'aura pas assisté à trois réunions consécutives, pourra être considéré comme démissionnaire.
+Les membres du **collège** sont élus pour une durée de un an et les membres sortants sont rééligibles.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+Les membres du **collège** sont élus pour une durée de un an et les membres sortants sont rééligibles.
 
-Le Conseil d'Administration peut inviter à l'une de ses réunions un membre actif non élu ou un membre d'honneur, pour voix consultative.
+Il est convenu qu'un minimum de 3/5e des sièges du **collège** est réservé aux membres actifs.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+Ils tiennent une comptabilité régulière de toutes les opérations qu'ils effectuent et rendent compte à **l’Assemblée Générale** annuelle qui approuve leur gestion.
 
-**Le Président** représente l'association dans tous les actes de la vie civile. Il a, notamment, qualité pour ester en justice au nom de l'association.
+En cas d'absence, le **collège** pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+En cas d'absence, le **collège** pourvoit provisoirement au remplacement de ses membres. Il est procédé à leur remplacement définitif par la prochaine assemblée générale ordinaire. Les pouvoirs des membres ainsi élus prennent fin à l'époque où devrait normalement expirer le mandat des remplacés.
 
-**Le Secrétaire** est chargé de tout ce qui concerne la correspondance et les archives. Il rédige, diffuse et archive les procès verbaux des réunions et assemblées et, en général, toutes les écritures concernant le fonctionnement de l’association, à l’exception de celles qui concernent la comptabilité. Il tient le registre spécial prévu par l’article 5 de la loi du 1er juillet 1901 et les article ...
+<!-- Prévoir ou non un vote pour élire les membres du collège? -->
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
-Le Conseil d'Administration peut inviter à l'une de ses réunions un membre actif non élu ou un membre d'honneur, pour voix consultative.
+Il est convenu qu'un minimum de 3/5e des sièges du **collège** est réservé aux membres actifs.
+
+Est éligible toute personne physique qui, au moment de l'envoi de la première convocation à l'élection, est un membre actif à jour de sa cotisation annuelle et âgé de 18 ans ou plus.
```
```suggestion
```

lutangar commented on README.md:
```diff
@@ -62,73 +63,92 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
+
+Est éligible toute personne physique qui, au moment de l'envoi de la première convocation à l'élection, est un membre actif à jour de sa cotisation annuelle et âgé de 18 ans ou plus.
+
+En cas de vacance (démission, radiation, décès...), le **collège** peut pourvoir provisoirement au remplacement des membres partants. Il est procédé à leur remplacement définitif lors de la prochaine Assemblée Générale. Les pouvoirs des membres ainsi désignés prennent fin à l'époque où devrait normalement expirer le mandat des membres remplacés.
```
```suggestion
```

yvalentin commented on README.md:
```diff
@@ -62,73 +63,76 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
 Les réunions font l'objet d'un procès-verbal.
 
 ## Article 13 – Assemblée Générale Ordinaire
 L'assemblée générale ordinaire comprend tous les membres à jour de leur cotisation. L'assemblée générale ordinaire se réunit chaque année. Quinze jours au moins avant la date fixée, les membres de l'association sont convoqués par les soins du secrétaire.
```
`par les soins du secrétaire.`

par les soins d'un membre designé par le collège

dolemoine replied:
```suggestion
L'assemblée générale ordinaire comprend tous les membres à jour de leur cotisation. L'assemblée générale ordinaire se réunit chaque année. Quinze jours au moins avant la date fixée, les membres de l'association sont convoqués 
par les soins d'un membre designé par le collège
```

yvalentin commented on README.md:
```diff
@@ -62,73 +63,76 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
 
-L'ordre du jour est indiqué sur les convocations. Le Président, assisté des membres du bureau, préside l'assemblée et expose la situation morale de l'association. Le trésorier rend compte de sa gestion et soumet le bilan à l'approbation de l'assemblée. Il est procédé, après épuisement de l'ordre du jour, au remplacement, au scrutin secret des membres du conseil sortant.
+L'ordre du jour est indiqué sur les convocations. Un membre désigné par le **collège**, préside l'assemblée et expose la situation morale de l'association.
+Le ou les membres délégataires de la signature sur le compte bancaire rendent compte de leurs gestion et soumet le bilan à l'approbation de l'assemblée. Il est procédé, après épuisement de l'ordre du jour, au remplacement, au scrutin secret des membres du conseil sortant.
```
`au scrutin secret des membres du conseil sortant`

au scrutin secret des membres du collège

yvalentin commented on README.md:
```diff
@@ -62,73 +63,76 @@ Sont membres adhérents ceux qui sont à jour de leur cotisation.
 Ne devront être traitées, lors de l'assemblée générale, que les questions soumises à l'ordre du jour.
 
-Dans le cas du renouvellement du bureau, l'ensemble des documents manipulés par l'association et nécessaires à son fonctionnement devront parvenir aux nouveaux membres du conseil dans un délais de trois semaines après leur nomination.
+Dans le cas du renouvellement du **collège**, l'ensemble des documents manipulés par l'association et nécessaires à son fonctionnement devront parvenir aux nouveaux membres du conseil dans un délais de trois semaines après leur nomination.
```
` Dans le cas du renouvellement du **collège**,`

y a t-il un renouvellement du collège ? Vu que celui-ci est composé "des membres actifs"

feugy replied:
Pas de renouvellement, on supprime donc le paragraphe

feugy committed:
Update README.md

Co-authored-by: dolemoine <43757076+dolemoine@users.noreply.github.com>

feugy committed:
Mise à jour suite à l'AGE

Merge des modifications apportées  par PierreM sur le [doc](https://docs.google.com/document/d/1lqhAf2RKDx6m1xTWNd-CY4HPu7OTzMzF8nhz8Tj5OUs/edit) ainsi que celle décidée dans le [CR](https://docs.google.com/document/d/1Qc1kapmBc-yxfd1jOF8Sm9ZD4pUn2dlICU44yMfjKBw/edit)

feugy committed:
Il n'y a plus de secrétaire

lutangar commented on README.md:
```diff
@@ -34,44 +34,41 @@ L’association est constituée pour une durée indéterminée.
-Le décès
-La radiation, prononcée par le **collège** pour non-paiement de la cotisation.
-L'exclusion, prononcée par le **collège** pour infraction aux présents statuts ou pour tout autre motif portant préjudice aux intérêts moraux et matériels de l'association.
+1. la démission adressée par écrit ou par mél au **collège**.
```
@feugy je ne suis pas convaincu de l'usage d'une liste ordonnée sur cette énumération. Difficile de statuer sur un ordre qui serait plus pertinent qu'un autre. De plus ce n'était pas l'object initial de ce commit ;)

feugy replied:
oh, pardon, absoluement. je vais remettre la liste à puce

feugy committed:
switch from numbered to bullet point list

lutangar commented on README.md:
```diff
@@ -19,116 +20,114 @@ L'objectif de cet espace est de:
 
-L'ordre du jour est indiqué sur les convocations. Le Président, assisté des membres du bureau, préside l'assemblée et expose la situation morale de l'association. Le trésorier rend compte de sa gestion et soumet le bilan à l'approbation de l'assemblée. Il est procédé, après épuisement de l'ordre du jour, au remplacement, au scrutin secret des membres du conseil sortant.
+L'ordre du jour est indiqué sur les convocations. Un membre désigné par le **collège**, préside l'assemblée et expose la situation morale de l'association.
+Le ou les membres délégataires de la signature sur le compte bancaire rendent compte de leur gestion et soumettent le bilan à l'approbation de l'assemblée. Il est procédé, après épuisement de l'ordre du jour, au remplacement, au scrutin secret des membres du **collège**.
```
```suggestion
Le ou les membres délégataires de la signature sur le compte bancaire rendent compte de leur gestion et soumettent le bilan à l'approbation de l'assemblée.
```

lutangar replied:
@yvalentin ok pour toi ?
